ui/app/mainui/AppMain.qml

```diff
@@ -1500,7 +1501,8 @@ Item {
             function open(address = "") {
                 if (!!address) {
                     preSelectedRecipient = address
-                    preSelectedRecipientType = TabAddressSelectorView.Type.Address
+                    // TO REVIEW!!
+                    preSelectedRecipientType = 0//Helpers.RecipientAddressObjectType.Address
```

Comment:
Please, some help on this! I'm sure it's nonsense but I don't see why it doesn't recognise the `Helpers.RecipientAddresObjectType` here..

Reply:
Not sure, you seem to have added `import shared.popups.send.panels 1.0` to get the enum but it lives in `import shared.popups.send 1.0` which is already imported... :shrug:

Reply:
I have no idea why it's not visible there :exploding_head: I tested it locally and the same problem here!

Reply:
@alexjba @Seitseman some ideas for this particular issue??

Reply:
@noeliaSD I will run qml debugger and enable more logging from Qt internals, maybe it will shed some light on this

Reply:
Hey, solved with @alexjba suggestion!! Added qualified import to the send popups import!! Thank you!!